CalWebViewApp/ios-webview-test-app.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this from the git index.  Add an entry in the .gitignore.

Reply:
Done